src/services/api/persona/index.ts

```diff
@@ -0,0 +1,118 @@
+import EnvironmentVariables from 'helpers/EnvVariables';
+import { TPersonaUser, TPersonaUserCreate, TPersonaUserFetch, TPersonaUserUpdate } from './models';
+import { sendRequest } from 'services/api';
+import { omit } from 'lodash';
+
+export const PERSONA_API_URL = `${EnvironmentVariables.configFor('PERSONA_API')}`;
+
+const DEFAULT_FIELDS_SELF = `
+  _id
+  title
+  firstName
+  lastName
+  egoId
+  roles
+  acceptedTerms
+  email
+  institutionalEmail
+  eraCommonsID
+  department
+  story
+  bio
+  jobTitle
+  institution
+  addressLine1
+  addressLine2
+  city
+  state
+  country
+  zip
+  phone
+  website
+  googleScholarId
+  twitter
+  facebook
+  github
+  linkedin
+  orchid
+  interests
+  acceptedKfOptIn
+  acceptedNihOptIn
+  acceptedDatasetSubscriptionKfOptIn
+  isPublic
+  isActive
+  hashedEmail
+`;
+
+export const headers = () => ({
```

Comment:
object would be less expensive than a func

Reply:
It was kinda standard on the includes project, I also prefer the object approach